.github/workflows/nodeCI.yml

```diff
@@ -4,14 +4,12 @@ on: [push, pull_request]
 
 jobs:
   build:
-
+    if: github.ref != 'refs/heads/master'
```

Comment:
any reason not to run this during merge to master?

Reply:
yes, if we merged to master it means that the PR we just merged already ran it. Also, even if it will fail the version will be created already. and during creation of a version we already run linter. Actually I don't think that workflow is useful as we don't have concrete logic, running it as part of PRs seems good enough.

Reply:
👍